ipaserver/install/server/install.py

```diff
@@ -336,6 +336,12 @@ def install_check(installer):
             "Please uninstall it before configuring the IPA server, "
             "using 'ipa-client-install --uninstall'")
 
+    instance_name = "-".join(options.realm_name.split("."))
+    if (is_ipa_configured() and
```

Comment:
If it is not started by the installer, then one needs to do it manually. Context from the original bug:
```
This causes problems especially in unattended container installations when completely new container is run for the second phase and the dirsrv is thus not up.
```
From this perspective it looks better to start it. 

From top of my head I cannot tell if there can be similar issue with other service we might depend on.